css/forgetpassword.css

```diff
@@ -0,0 +1,24 @@
+
+body{
+  background:rgba(196,196,196,0.17);
+}
+
+.display{
+  height:200px;
+  background:#1F59BB;
+}
+
+.form-container{
+  position:absolute;
+  padding:30px;
+  background:#ffffff;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#ffffff" to be "#fff" (color-hex-length)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5538575)